config_cmd.go

```diff
@@ -97,11 +97,12 @@ Per-Feed Configuration Options
 
 Key           | Purpose
 --------------+--------------------------------------------------------------
-delay         | The amount of time to sleep between retried HTTP-fetches.
+delay         | The amount of time to sleep between retrying a failed HTTP-fetch.
```

Comment:
I know that this option is expressed in seconds, but maybe it would be good to set it explicit?

Reply:
Good comment, thank you.  Updated :)